source/numerics.tex

```diff
@@ -1912,21 +1912,21 @@
     with the same initial state
     as all other default-constructed engines
     of type \tcode{E}.
-  & \bigoh{$\mbox{size of state}$}
```

Comment:
Beats me, I tried reading https://tex.stackexchange.com/questions/70632/difference-between-various-methods-for-producing-text-in-math-mode but it gave me a headache :)